src/Utils/DPop.php

```diff
@@ -310,9 +346,6 @@ public function validateDpop($dpop, $request) {
 			throw new InvalidTokenException("jti is invalid");
 		}
 
-		// 10. that, if used with an access token, it also contains the 'ath' claim, with a hash of the access token
-		// TODO: implement
-
 		return true;
```

Comment:
Return value can be removed as it never makes sense to check it. (Also currently the doc-block lies).

Reply:
same here